js/twitterApi.js

```diff
@@ -60,13 +66,65 @@ $(document).ready(function(){
 
 			// So that no person is selected more than once
 			twitterUsersArr.splice(rand, 1);
-
-			console.log(twitterUsersArr.length);
 		}
 
```

Comment:
I think we should get rid of all this unnecessary whitespace. Let me know if you're handling it, or if you want I'll merge it then take care of it.